src/uikit.cs

```diff
@@ -10614,6 +10622,29 @@ interface UIResponder : UIAccessibilityAction, UIAccessibilityFocus, UIUserActiv
 		[TV (15,0), iOS (15,0), MacCatalyst (15,0)]
 		[Export ("captureTextFromCamera:")]
 		void CaptureTextFromCamera ([NullAllowed] NSObject sender);
+
+		[NoWatch, NoTV, NoiOS]
+		[MacCatalyst (13, 0)]
+		[Export ("childViewControllerForTouchBar")]
+		[NullAllowed]
+		UIViewController ChildViewControllerForTouchBar { get; }
+
+		[NoWatch, NoTV, NoiOS]
+		[MacCatalyst (13,0)]
+		[Export ("setNeedsTouchBarUpdate")]
+		void SetNeedsTouchBarUpdate ();
+
+#if __MACCATALYST__
+		[MacCatalyst (13, 0)]
+		[Export ("makeTouchBar")]
+		[return: NullAllowed]
+		NSTouchBar CreateTouchBar ();
+
+		[MacCatalyst (13, 0)]
+		[Export ("touchBar", ArgumentSemantic.Strong)]
+		[NullAllowed]
+		NSTouchBar TouchBar { get; set; }
+#endif
```

Comment:
You can do the using to be NSObject, then use [NoXYZ] which is simpler to parse than #if. I would just use #if it they are due to some struct or manual code in not processed file.